src/symbolic/symbolic_memory_concretizing.ml

```diff
@@ -113,11 +113,15 @@ module Backend = struct
       match Hashtbl.find m.chunks base with
       | exception Not_found -> return (Error Trap.Memory_leak_use_after_free)
       | size ->
-        let+ is_out_of_bounds = select (I32.ge_u offset size) in
+        let upper_bound = I32.add offset (const_i32 (Int32.of_int n)) in
```

Comment:
Calling this `upper_bound` is a little bit confusing as I was expecting it to be the bound that should not be crossed. It is actually the last element that will be written, right? What about naming it `last` (or anything better if you have an idea) ?

Reply:
Yes I agree, maybe `last_offset`?